Templates/cmake/CXX/IPO/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#==============================================================
```

Comment:
C and CXX cmake files are identical?

Reply:
The files are not identical but very similar. The main intent is to show the usage of cmake with icx/icpx. Also, I wanted each sub directory self contained to be able to compile by itself.